frontend/pages/SoftwarePage/components/forms/SoftwareAndroidForm/SoftwareAndroidForm.tsx

```diff
@@ -0,0 +1,227 @@
+        }
+      : {
+          applicationID: "",
+          selfService: true, // Default to true for new Android apps
+          automaticInstall: false, // 4.77 Currently navailable for Android apps
+          targetType: "All hosts",
+          customTarget: "labelsIncludeAny",
+          labelTargets: {},
+          categories: [],
+          platform: "android",
+        }
+  );
+
+  const [formValidation, setFormValidation] = useState<IFormValidation>({
+    isValid: !!softwareAndroidForEdit, // Disables submit before Android application ID is entered
+  });
+
+  const onFormSubmit = (evt: React.FormEvent<HTMLFormElement>) => {
+    evt.preventDefault();
+    onSubmit(formData);
+  };
+
+  const onInputChange = ({ name, value }: IInputFieldParseTarget) => {
+    const newFormData = { ...formData, [name]: value };
+    setFormData(newFormData);
+    setFormValidation(generateFormValidation(newFormData));
+  };
+
+  const onToggleSelfServiceCheckbox = (value: boolean) => {
+    const newData = { ...formData, selfService: value };
+    setFormData(newData);
+  };
+
+  const onSelectCategory = ({
+    name,
+    value,
+  }: {
+    name: string;
+    value: boolean;
+  }) => {
+    let newCategories: string[];
+
+    if (value) {
+      // Add the name if not already present
+      newCategories = formData.categories.includes(name)
+        ? formData.categories
+        : [...formData.categories, name];
+    } else {
+      // Remove the name if present
+      newCategories = formData.categories.filter((cat) => cat !== name);
+    }
+
+    const newData = {
+      ...formData,
+      categories: newCategories,
+    };
+
+    setFormData(newData);
+    setFormValidation(generateFormValidation(newData));
+  };
+
+  const onToggleAutomaticInstall = (value: boolean) => {
+    const newData = { ...formData, automaticInstall: value };
+    setFormData(newData);
+  };
+
+  const isSubmitDisabled = !formValidation.isValid;
+
+  const renderContent = () => {
+    // Edit Android does not exist on 4.77
+    if (softwareAndroidForEdit) {
+      return null;
+    }
+
+    // Add Android form
+    return (
+      <div className={`${baseClass}__form-fields`}>
+        <InputField
+          autoFocus
+          label="Application ID"
+          placeholder="com.android.chrome"
+          helpText={
+            <>
+              The ID at the end of the app&apos;s{" "}
+              <CustomLink
+                text="Google Play URL"
+                url={`${LEARN_MORE_ABOUT_BASE_LINK}/google-play-store`}
+                newTab
+              />{" "}
+              E.g. &quot;com.android.chrome&quot; from
+              &quot;https://play.google.com/store/apps/details?id=com.android.chrome&quot;
+            </>
+          }
+          onChange={onInputChange}
+          name="applicationID"
+          value={formData.applicationID}
+          parseTarget
+          disabled={gitOpsModeEnabled} // TODO: Confirm GitOps behavior
```

Comment:
makes sense I left this as a TODO since this ticket didnt talk about it, but I now realized there's a separate ticket for this.